src/wp-includes/class-wp-scripts.php

```diff
@@ -911,6 +955,10 @@ public function in_default_dir( $src ) {
 	 * @return bool True on success, false on failure.
 	 */
 	public function add_data( $handle, $key, $value ) {
+		if ( ! array_key_exists( $handle, $this->registered ) ) {
```

Comment:
https://github.com/WordPress/wordpress-develop/pull/4391/files?file-filters%5B%5D=.php&show-viewed-files=true#r1232052943

Reply:
Done in 47b2cbccfc7641d141994c5ec0a28e356d12da3e